openassessment/xblock/static/js/src/lms/oa_peer.js

```diff
@@ -123,9 +152,14 @@ OpenAssessment.PeerView.prototype = {
                 // Override default form submission
                 eventObject.preventDefault();
 
+
+                //Status will change in update announce it to the Screen Reader after Render
+                view.announceStatusChange = true;
```

Comment:
This call sets the flag on a submit to announce the status change after the render is complete.